src/pudl/transform/eia860.py

```diff
@@ -158,9 +158,24 @@ def _core_eia860__ownership(raw_eia860__ownership: pd.DataFrame) -> pd.DataFrame
     state_to_country = {
         x.subdivision_code: x.country_code for x in POLITICAL_SUBDIVISIONS.itertuples()
     } | {"CN": "CAN"}
-    own_df["owner_country"] = own_df["owner_state"].map(state_to_country)
+    own_df["country"] = own_df["owner_state"].map(state_to_country)
```

Comment:
Not a big deal at all, but I might leave this as `owner_country`, then rename below with all the other columns just for consistency.